service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -84,13 +85,13 @@ private void initSparkService() {
         // But don't forget, a load balancer may call this URL very often (like 1 per sec),
         // so don't make this request processor heavy, or implement is as independent background service.
         LOG.info("Register /health URL");
+        LOG.info("Use /health?pretty to pretty-print output JSON");
         Spark.get("/health", (req, res) -> {
             res.status(HttpStatusCode.OK_200);
             res.type(ContentType.APPLICATION_JSON.getMimeType());
 
-            return healthResponse;
-
-        }, SphereJsonUtils::toJsonString);
+            return req.queryParams("pretty") != null ? toPrettyJsonString(healthResponse) : toJsonString(healthResponse);
```

Comment:
`//For now it just returns a JSON response {"status":200}` <- This comment is not valid anymore right? The JSON returned contains information about the tenants and the application info.

Reply:
d098fc76